css-view-transitions-1/Overview.bs

```diff
@@ -1320,7 +1320,8 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 		1. Set |transition|'s [=ViewTransition/initial snapshot containing block size=] to the [=snapshot containing block size=].
 
 		1. [=list/For each=] |element| of every [=/element=] that is [=/connected=],
-			and has a [=node document=] equal to to |document|,
+			and has a [=node document=] equal to |document|,
+			and has a [=tree context=] equal to |document|,
```

Comment:
Doesn't "capture the new state" also have a similar loop that should use the same check?

Reply:
Nice catch, done!